airbrakes/data_handling/imu_data_packet.py

```diff
@@ -26,6 +26,18 @@ class RawDataPacket(IMUDataPacket):
     scaledGyroX: float | None = None
     scaledGyroY: float | None = None
     scaledGyroZ: float | None = None
+    # deltaVel units are in g seconds
+    deltaVelX: float | None = None
+    deltaVelY: float | None = None
+    deltaVelZ: float | None = None
+    # TODO: check if these exist
+    deltaThetaX: float | None = None
+    deltaThetaY: float | None = None
+    deltaThetaZ: float | None = None
+    # mag units are in gauss, they can be used in addition to the accelerometer to determine orientation
+    stabilizedMagX: float | None = None
+    stabilizedMagY: float | None = None
+    stabilizedMagZ: float | None = None
```

Comment:
pretty sure these do not exist. We do not have a magnetometer.

Reply:
they will need to be tested, but sensor connect said it did